docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,32 +1,201 @@
+
+
+### Step 3. Add Code
+
+Add code to perform the following actions:
+
+1. Create a collection called `ddos` to which we will subscribe. If a collection by that name already exists, the existing collection is used instead.
+1. Add `data` to the collection, then subscribe to the collection. In this example, we are adding IP addresses to block.
+1. Delete `data` from the collection. If the collection becomes empty, the collection is deleted too.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="JavaScript">
+
+```js
+async function main () {
+	async function createCollection () {
+		console.log("\n1. Log in.");
+		console.log("\n2. Create collection.");
+		try {
+		  console.log(`Creating the collection ${collectionName}...`);
+		  const existsColl = await client.hasCollection(collectionName);
+		  if (existsColl === false) {
+			await client.createCollection(collectionName, { stream: true });
+      console.log(`Collection created`);
+		  }
+		  // Add an onChange listener for collection
+		  listener = await client.onCollectionChange(collectionName);
+
+		  // Decode the message printed here in readable format
+		  listener.on("message", (msg) => {
+			const receivedMsg = msg && JSON.parse(msg);
+			console.log("message=>", Buffer.from(receivedMsg.payload, "base64").toString("ascii"))
+		  });
+		  listener.on("open", () => console.log("Connection open"));
+		  listener.on("close", () => console.log("Connection closed"));
+		} catch (e) {
+		  await console.log("Collection creation did not succeed due to " + e);
+		}
+	}
+	await createCollection();
+
+	async function insertData () {
+		console.log(`\n3. Insert data`);
+		await client.insertDocumentMany(collectionName, data);
+	}
+	await sleep(2000);
+	await insertData();
+
+	async function deleteData () {
+		console.log("\n4. Delete data");
+		await client.deleteCollection(collectionName);
+	}
+	await sleep(10000);
+	// Closing OnChange listener
+	await listener.close();
+	await deleteData();  
+}
+main();
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step1: Open connection to GDN. You will be routed to closest region.
+    print(f"\n1. CONNECT: federation: {URL}")
+    client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+    # Step 2: Create a collection if one does not exist
+    print(f"\n2. CREATE_COLLECTION: region: {URL},  collection: {COLLECTION_NAME}")
+    if client.has_collection(COLLECTION_NAME):
+        collection = client.collection(COLLECTION_NAME)
+    else:
+        collection = client.create_collection(COLLECTION_NAME, stream=True)
+
+    # Subscribe to be notified when changes are made to collection.
+
+    def create_callback():
+        def callback_fn(event):
+            pp.pprint(event)
+            return
+
+        client.on_change(COLLECTION_NAME, callback=callback_fn, timeout=15)
+
+    # Step3: Subscribe to receive documents in realtime (PUSH model)
```

Comment:
We don't receive documents exactly.. it should be rephrased I guess, we receive real-time updates/changes made to the collection.. it can be deleting a document or updating a document as well.